ERROR: No matching export in "node_modules/.pnpm/three@0.142.0/node_modules/three/build/three.module.js" for import "Mapping"

Hey there, first time trying skinview3d and when installing the latest version I get this error:
```
X [ERROR] No matching export in "node_modules/.pnpm/three@0.142.0/node_modules/three/build/three.module.js" for import "Mapping"

    node_modules/.pnpm/skinview3d@3.0.0/node_modules/skinview3d/libs/viewer.js:2:157:
      2 │ ...mera, Scene, Texture, Vector2, WebGLRenderer, AmbientLight, Mapping, CanvasTexture, WebGLRenderTarget, FloatType, DepthTexture,... 
        ╵                                                                ~~~~~~~

D:\Desktopbackup\Documents\Namer\node_modules\.pnpm\esbuild@0.18.20\node_modules\esbuild\lib\main.js:1649
  let error = new Error(`${text}${summary}`);
              ^

Error: Build failed with 1 error:
node_modules/.pnpm/skinview3d@3.0.0/node_modules/skinview3d/libs/viewer.js:2:157: ERROR: No matching export in "node_modules/.pnpm/three@0.142.0/node_modules/three/build/three.module.js" for import "Mapping"
    at failureErrorWithLog (D:\Desktopbackup\Documents\Namer\node_modules\.pnpm\esbuild@0.18.20\node_modules\esbuild\lib\main.js:1649:15)       
    at D:\Desktopbackup\Documents\Namer\node_modules\.pnpm\esbuild@0.18.20\node_modules\esbuild\lib\main.js:1058:25
    at D:\Desktopbackup\Documents\Namer\node_modules\.pnpm\esbuild@0.18.20\node_modules\esbuild\lib\main.js:1525:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  errors: [
    {
      detail: undefined,
      id: '',
      location: {
        column: 157,
        file: 'node_modules/.pnpm/skinview3d@3.0.0/node_modules/skinview3d/libs/viewer.js',
        length: 7,
        line: 2,
        lineText: 'import { Color, PointLight, EquirectangularReflectionMapping, Group, NearestFilter, PerspectiveCamera, Scene, Texture, Vector2, WebGLRenderer, AmbientLight, Mapping, CanvasTexture, WebGLRenderTarget, FloatType, DepthTexture, Clock, Object3D } from "three";',
        namespace: '',
        suggestion: ''
      },
      notes: [],
      pluginName: '',
      text: 'No matching export in "node_modules/.pnpm/three@0.142.0/node_modules/three/build/three.module.js" for import "Mapping"'
    }
  ],
  warnings: []
}

Node.js v20.6.1
```

For now I switched to 3.0.0-alpha.1 but I am wondering why this occurs.

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
 		"assets"
 	],
 	"dependencies": {
-		"@types/three": "^0.142.0",
+		"@types/three": "^0.156.0",
 		"skinview-utils": "^0.7.1",
 		"three": "^0.156.1"
 	},

--- a/src/viewer.ts
+++ b/src/viewer.ts
@@ -23,7 +23,7 @@ import {
 	Vector2,
 	WebGLRenderer,
 	AmbientLight,
-	Mapping,
+	type Mapping,
 	CanvasTexture,
 	WebGLRenderTarget,
 	FloatType,
@@ -32,7 +32,8 @@ import {
 	Object3D,
 } from "three";
 import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js";
-import { EffectComposer, FullScreenQuad } from "three/examples/jsm/postprocessing/EffectComposer.js";
+import { EffectComposer } from "three/examples/jsm/postprocessing/EffectComposer.js";
+import { FullScreenQuad } from "three/examples/jsm/postprocessing/Pass.js";
 import { RenderPass } from "three/examples/jsm/postprocessing/RenderPass.js";
 import { ShaderPass } from "three/examples/jsm/postprocessing/ShaderPass.js";
 import { FXAAShader } from "three/examples/jsm/shaders/FXAAShader.js";

